### PR TITLE
Thor/update rebasing

### PR DESCRIPTION
**Pull request by wikfeldt**
point out some negative aspects of rebasing, and extending the exercise with a bonus part

### CODE REVIEW

File: _episodes/01-rebase.md [hunk 1]
```diff
@@ -47,9 +50,10 @@ $ git rebase master
 ### Advantages and disadvantages
 
 - `git rebase` makes "merges" producing a linear history.
+- `git merge` resolves all conflicts in a single commit, with `git rebase` each commit being rebased needs 
+  conflict resolution.
```
Comment (bast) [inline _episodes/01-rebase.md:54]: "each commit may need" - only the conflicting commits need conflict resolution, not necessarily each commit.

File: _episodes/01-rebase.md [hunk 2]
```diff
@@ -47,9 +50,10 @@ $ git rebase master
...
 - `git rebase` may invalidate tests.
-- When somebody asks you to rebase your work to the work of somebody else you
-  know what this means (GitHub can do this automatically for you).
+- `git merge` creates history as it actually happened, `git rebase` shows a simpler linear history.
```
Comment (bast) [inline _episodes/01-rebase.md]: maybe:
- git merge preserves chronology of commits and creates explicit merge commits (unless fast-forward).
- git rebase can change chronology of commits

File: _episodes/01-rebase.md (hunk 1 above)
Comment (wikfeldt) [reply _episodes/01-rebase.md:54]: indeed, i'll fix it, thanks

File: _episodes/01-rebase.md (hunk 2 above)
Comment (wikfeldt) [reply _episodes/01-rebase.md]: i agree, fixed this as well. thanks for review

**Review by bast — APPROVED** [review]

**Pull request merged**